tests/python/unittest/test_operator.py

```diff
@@ -1659,7 +1658,7 @@ def test_depthwise_convolution():
                             exe2.backward(exe2.outputs[0])
```

Comment:
can we check whether a downcast is happening when we do `arr2[:] = arr1` from float64 to float32. If yes lets fix that check if the test is still flaky.

Reply:
arr1 and arr2 are both float32. Downcast is happening during populating arr1 by random normal samples. I have made the casting explicit just to avoid errors.